src/material/core/common-behaviors/common-module.ts

```diff
@@ -53,7 +54,7 @@ export interface GranularSanityChecks {
  * This module should be imported to each top-level component module (e.g., MatTabsModule).
  */
 @NgModule({
-  imports: [BidiModule],
+  imports: [A11yModule, BidiModule],
```

Comment:
I'm not sure whether we should be adding the `A11yModule` to all of the Material modules since it's one of our largest modules and we only need the high contrast detection. It might be better to just inject the `HighContrastModeDetector` since it's a tree-shakeable provider and add the call to set the body class. We can do the same in the `A11yModule` to cover users that import the CDK, but not Material.

Reply:
Done out of additional caution, but adding `A11yModule` _shouldn't_ increase any size if we don't actually consume any of the directives or injectables.